.env.example

```
@@ -23,6 +23,11 @@ WORKER_ENABLE=true
 WORKER_RETRIES=3
 WORKER_BACKOFF_S=5
 
+PROMETHEUS_PORT=9091
+GRAFANA_PORT=8000
+GRAFANA_ADMIN_USER=admin
+GRAFANA_ADMIN_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_PASSWORD key should go before the GRAFANA_ADMIN_USER key

(UnorderedKey)